daprdocs/content/en/getting-started/install-dapr-cli.md

```diff
@@ -16,13 +16,29 @@ This command installs the latest linux Dapr CLI to `/usr/local/bin`:
 ```bash
 wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash
 ```
+
+{{% alert title="Installation without `sudo`" color="primary" %}}
```

Comment:
Alert shortcodes don't work inside of tabs: 
![image](https://user-images.githubusercontent.com/54363786/125550073-f6fa3f37-acb3-4a29-8fc8-2e23e2472af9.png)

Reply:
Unfortunately can't make a suggestion since there's a ``` in the suggestion (pls let me know if there's an escape character)

Reply:
Then what do you want this to be @AaronCrawfis?

Reply:
The M1 Mac section is already broken today due to the use of Alert Codes.